pkg/runner/runner.go

```diff
@@ -287,6 +301,26 @@ func (r *Runner) deployAssets() error {
 			return err
 		}
 	}
+	if r.gatewayApi {
+		log.Debugf("Creating GatewayClass...")
+		_, err = hrClientSet.GatewayV1beta1().GatewayClasses().Create(context.TODO(), gatewayClass, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+		time.Sleep(5 * time.Second) // wait for ServiceMeshControlPlane to be ready
```

Comment:
ServiceMeshControlPlane?

Reply:
does it require OSSM?

Reply:
yes, our current proposal uses OSSM under the hood